packages/ses/src/module-link.js

```diff
@@ -129,33 +123,31 @@ export const instantiate = (
     return mapGet(instances, moduleSpecifier);
   }
 
-  validateStaticModuleRecord(staticModuleRecord, moduleSpecifier);
+  validateStaticModuleRecord(moduleSource, moduleSpecifier);
 
   const importedInstances = new Map();
   let moduleInstance;
-  if (isPrecompiled(staticModuleRecord)) {
-    validatePrecompiledStaticModuleRecord(staticModuleRecord, moduleSpecifier);
+  if (isPrecompiled(moduleSource)) {
+    validatePrecompiledStaticModuleRecord(moduleSource, moduleSpecifier);
     moduleInstance = makeModuleInstance(
       compartmentPrivateFields,
       moduleAliases,
       moduleRecord,
       importedInstances,
     );
-  } else if (isThirdParty(staticModuleRecord)) {
-    validateThirdPartyStaticModuleRecord(staticModuleRecord, moduleSpecifier);
+  } else if (isThirdParty(moduleSource)) {
+    validateThirdPartyStaticModuleRecord(moduleSource, moduleSpecifier);
     moduleInstance = makeThirdPartyModuleInstance(
       compartmentPrivateFields,
-      staticModuleRecord,
+      moduleSource,
       compartment,
       moduleAliases,
       moduleSpecifier,
       resolvedImports,
     );
   } else {
     throw TypeError(
-      `importHook must return a static module record, got ${q(
-        staticModuleRecord,
-      )}`,
+      `importHook must return a static module record, got ${q(moduleSource)}`,
```

Comment:
Did you mean?

```suggestion
      `importHook must return a static module source, got ${q(moduleSource)}`,
```

Reply:
I’ve added a commit that updates all the terminology:

- “module records” now always refer to the internal record for a module instance
- “static module records” are now “module sources”
- “third party static module records” are now “virtual module sources“

This generally makes “module record” much less confusing, so a clarifying comment is no longer necessary at the head of `module-load.js` and `module-link.js`.

Reply:
Resolving